sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -1012,6 +1017,7 @@ export class MessageSession extends LinkEntity {
         );
 
         this._receiver = await this._context.namespace.connection.createReceiver(options);
+        this.isClosedDueToExpiry = false;
```

Comment:
This is initialized to `false` in the constructor. Any reason to set to `false` again here?

Reply:
Please see other comment